src/components/Table/TableBody/TableBodyRow/TableBodyRow.jsx

```diff
@@ -94,6 +94,8 @@ const propTypes = {
   learnMoreText: PropTypes.string,
   /** I18N label for dismiss */
   dismissText: PropTypes.string,
+  /** table toolbar */
+  editBar: PropTypes.oneOf(['buttons', 'toast']),
```

Comment:
This name is confusing to me. Is it a state of the toolbar? I'm unclear what buttons vs toast is. I think there may be a more appropriate name and value options.

Reply:
Hi, 'buttons' means toolbar come to save/cencel buttons status, besides 'toast' means toolbar come to toast status. It's a state of the toobar. Maybe we can change these to 'edit' and 'undo' or whatever, what do you recommend?
![image](https://user-images.githubusercontent.com/37580544/71044346-600ef900-216c-11ea-98e6-96a9290b0c37.png)
![image](https://user-images.githubusercontent.com/37580544/71044356-67ce9d80-216c-11ea-8608-f1f1c472e4e1.png)

Reply:
And, we use callback (hook) to write the codes, so I didn't put editBar with activeBar together.

Reply:
I don't think that 'toast' is something that the table action bar should manage.  A toast is something that happens in response to the action (save or cancel) and that is generally up to the application to supply.   I'm not ever sure how I'd use this prop value... are you suggesting that I first set it to buttons and then hook into the on save and then set it to `taost`??